### PR TITLE
Add Contributors Table in README.md

### PR DESCRIPTION
**Pull request by samir-0711**
Added contributors table in README.md
Issue #29 

Original
image

Updated
image

### CODE REVIEW

**Comment by IIITM-Jay** [comment]
@samir-0711 Samir I think this is the awesome work you have done :1st_place_medal: :champagne: :sparkles: :congratulations:

**Comment by IIITM-Jay** [comment]
@samir-0711  Samir can we have other icons as bug fixer, technical writer... 
I mean the caterpillar/ bug icon, a pen for technical writer, an eye for code reviewers...
I think that will go to the even higher level and I will tag it accordingly

**Comment by samir-0711** [comment]
OK, I am changing icons in this pr

**Comment by samir-0711** [comment]
Requested changes done @IIITM-Jay

File: README.md [hunk 1]
```diff
@@ -24,3 +24,55 @@ The project deals with managing different layers of property on the basis of dif
...
+
+Thanks goes to these wonderful people ✨✨:
+<table>
+	<tr>
+		<td align="center">
+			<a href="https://github.com/IIITM-Jay">
+				<img src="https://avatars2.githubusercontent.com/u/65283880?v=4" width="100px" alt="" />
+				<br /> <sub><b>IIITM-Jay</b></sub>
+			</a>
+			<br /> <a href="https://github.com/Jayshah6699/datascience-mashup/commits?author=IIITM-Jay">
```
Comment (IIITM-Jay) [inline README.md:37]: is the space under ```<br />``` is required...

**Review by IIITM-Jay — COMMENTED** [review]
Awesome Work Samir! @samir-0711  I have single doubt to clear with you and it's ready to merge now. :sparkles: :champagne:

**Review by Nandini-Jha — COMMENTED** [review]
Samir @samir-0711 actually I don't have enough words to express your work. This really look nice and pretty. @IIITM-Jay  It's ready to get merged.

Welcome once again Samir as a new contributor to this project and repo :champagne:

File: README.md (hunk 1 above)
Comment (samir-0711) [reply README.md:37]: It is not required. If you want I can remove the spaces.

Comment (samir-0711) [reply README.md]: I have removed the spaces @IIITM-Jay

**Comment by samir-0711** [comment]
Can you please change "DWoC" label to "DWOC" @IIITM-Jay @Nandini-Jha 
Thank You

**Comment by IIITM-Jay** [comment]
@all-contributors please add @samir-0711 for design,documentation

**Comment by IIITM-Jay** [comment]
@all-contributors please add @samir-0711 for design, documentation

**Comment by IIITM-Jay** [comment]
@all-contributors please add @samir-0711 for documentation

**Pull request merged**